client/lcd/test_helpers.go

```diff
@@ -114,7 +114,15 @@ func CreateAddr(t *testing.T, name, password string, kb crkeys.Keybase) (sdk.Acc
 // their respective sockets where nValidators is the total number of validators
 // and initAddrs are the accounts to initialize with some steak tokens. It
 // returns a cleanup function, a set of validator public keys, and a port.
-func InitializeTestLCD(t *testing.T, nValidators int, initAddrs []sdk.AccAddress) (func(), []crypto.PubKey, string) {
+func InitializeTestLCD(
+	t *testing.T, nValidators int, initAddrs []sdk.AccAddress,
+) (cleanup func(), valPubKeys []crypto.PubKey, valOperAddrs []sdk.ValAddress, port string) {
```

Comment:
This shouldn't just be `valPubKey` should be either `valConsPubKey` or `valOperPubKey`